flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionJobVertex.java

```diff
@@ -499,31 +497,6 @@ public void fail(Throwable t) {
         }
     }
 
-    public void resetForNewExecution(final long timestamp, final long expectedGlobalModVersion)
```

Comment:
I think `ExecutionVertex#resetForNewExecution(final long timestamp, final long originatingGlobalModVersion)` was used by this method. We should remove it as well.
Besides that, could we have a separate hotfix commit to remove this method because it is not the work of removing globalModVersion?